src/main/java/com/basho/riak/client/core/operations/ts/DeleteOperation.java

```diff
@@ -0,0 +1,119 @@
+package com.basho.riak.client.core.operations.ts;
+
+import com.basho.riak.client.core.converters.TimeSeriesPBConverter;
+import com.basho.riak.client.core.operations.PBFutureOperation;
+import com.basho.riak.client.core.query.timeseries.Cell;
+import com.basho.riak.client.core.util.BinaryValue;
+import com.basho.riak.protobuf.RiakKvPB;
+import com.basho.riak.protobuf.RiakMessageCodes;
+import com.google.protobuf.ByteString;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+/**
+ * An operation to delete a row in a Riak Time Series table.
+ *
+ * @author Alex Moore <amoore at basho dot com>
+ * @since 2.0.3
+ */
+public class DeleteOperation extends PBFutureOperation<Void, RiakKvPB.TsDelResp, BinaryValue>
+{
+    private static final Logger logger = LoggerFactory.getLogger(DeleteOperation.class);
+    private final Builder builder;
+
+    private DeleteOperation(Builder builder)
+    {
+        super(RiakMessageCodes.MSG_TsDelReq,
+              RiakMessageCodes.MSG_TsDelResp,
+              builder.reqBuilder,
+              RiakKvPB.TsDelResp.PARSER);
+
+        this.builder = builder;
+    }
+
+    @Override
+    protected Void convert(List<RiakKvPB.TsDelResp> responses)
+    {
+        // This is not a streaming op, there will only be one response
+        if (responses.size() > 1)
+        {
+            logger.error("Received {} responses when only one was expected.", responses.size());
+        }
+
+        return null;
+    }
+
+    @Override
+    public BinaryValue getQueryInfo()
```

Comment:
We get more than 1 response in the case of streaming ops.  I'd have to check if we get others anyplace else.